src/batched/KokkosBatched_Util.hpp

```diff
@@ -24,42 +25,43 @@ namespace KokkosBatched {
 #define Int2StringHelper(A) #A
 #define Int2String(A) Int2StringHelper(A)
 #define StringCat(A,B) A B
-      
-#if defined(KOKKOS_ENABLE_PRAGMA_UNROLL)
-#pragma message "KOKKOS_ENABLE_PRAGMA_UNROLL"
-#else
-#pragma message "KOKKOS_DISABLE_PRAGMA_UNROLL"
-#endif
 
-#if defined(__AVX__) || defined(__AVX2__)
-#pragma message "KOKKOS_BATCHED_ENABLE_AVX256"
-#else
-#pragma message "KOKKOS_BATCHED_DISABLE_AVX256"
-#endif
+    void print_compiler_info();
 
-#if defined(__AVX512F__)
-#pragma message "KOKKOS_BATCHED_ENABLE_AVX512"
-#else
-#pragma message "KOKKOS_BATCHED_DISABLE_AVX512"
-#endif
+    // we do not allow mixed precision
+    template<typename Ta, typename Tb>
+    struct is_convertible {
```

Comment:
Calling this `is_convertible` will break user code that does `using namespace std;`.  I know users aren't supposed to do that, but they do :-) .